services/catalog/src/simcore_service_catalog/services/manifest.py

```diff
@@ -80,18 +83,43 @@ async def get_services_map(
     return services
 
 
+@cached(
+    ttl=DIRECTOR_CACHING_TTL,
+    namespace=__name__,
+    key_builder=lambda f, *ag, **kw: f"{f.__name__}/{kw['key']}/{kw['version']}",
+)
```

Comment:
The decorator has some defaults. This is where you configure it

Reply:
@bisgaard-itis this is time-constraint ... (ttl=time-to-live)
I am not sure about the memory though (i.e. how many items it caches). 
I wonder what happens in a memory-bound countainer. I will test and probably add a limit to avoid the app from getting killed by the orchestrator